packages/styles/tabs.css

```diff
@@ -26,14 +26,13 @@
   border-left: 1px solid var(--tabs-border-color);
 }
 
+/* Texts that are longer the Tab's width will be filtered by design group */
```

Comment:
I don't quite understand this comment. Since this is a public facing component library, this may be used by more than just deque.

Reply:
Good point